components/header/index.tsx

```diff
@@ -0,0 +1,24 @@
+"use client"
+
+import { usePathname } from "next/navigation"
+import Link from "next/link"
+
+export const Header = () => {
+  const pathname = usePathname()
+  return (
+    <header className="bg-sky-400 text-white font-bold h-8 pl-5">
+      <Link
+        className={`link ${pathname === "/about" ? "underline" : ""}`}
```

Comment:
I suggest you to use `clsx` + `tailwind-merge`. Create a `cn` function in `utilities/cn.ts`. Use the same code as in [this example](https://youtu.be/re2JFITR7TI?si=Gw-FEzh_EKBYqBB-).